### PR TITLE
SQL: forced versioning global variables [closes #32]

### LINKED ISSUE #32: Add "forced versioning" option

### PR DESCRIPTION
**Pull request by kevgs**
Set variables like this:
./mtr --mem --suite=main --mysqld='--force-versioning=1' --mysqld='--force-hidden=1'

### CODE REVIEW

**Comment by midenok** [comment]
Wrong base.

File: sql/sys_vars.cc [hunk 1]
```diff
@@ -387,6 +387,16 @@ static Sys_var_charptr sys_temporal_current_timestamp(
        GLOBAL_VAR(temporal_current_timestamp), CMD_LINE(REQUIRED_ARG, 'b'),
        IN_FS_CHARSET, DEFAULT("now"));
 
+static Sys_var_mybool Sys_force_versioning(
+       "force_versioning", "Makes all creating tables versioned by default",
```
Comment (midenok) [inline sql/sys_vars.cc:391]: "Force system versioning for all created tables"

File: sql/sys_vars.cc [hunk 2]
```diff
@@ -387,6 +387,16 @@ static Sys_var_charptr sys_temporal_current_timestamp(
...
+       GLOBAL_VAR(force_versioning), CMD_LINE(OPT_ARG), DEFAULT(FALSE));
+
+static Sys_var_ulong Sys_force_hidden(
+       "force_hidden", "Make versioning info hidden from everything "
```
Comment (midenok) [inline sql/sys_vars.cc:395]: Sys_hide_versioning
"hide_versioning", "Hide system versioning from being displayed in table info"

File: sql/sys_vars.cc [hunk 3]
```diff
@@ -387,6 +387,16 @@ static Sys_var_charptr sys_temporal_current_timestamp(
...
+       "(2 - not specified, 0 - not hidden, 1 - hidden)",
```
Comment (midenok) [inline sql/sys_vars.cc:396]: **2**: needless for user. Should be ON/OFF.

**Review by midenok — CHANGES_REQUESTED** [review]
Please add plugin files, make `hide_versioning` bool, etc (see comments).

File: sql/sys_vars.cc (hunk 3 above)
Comment (kevgs) [reply sql/sys_vars.cc:396]: It's not unuseful. If you not specify `force_hidden` and specify `force_versioning` you'll get versioning + hidden. If we make this `bool` we'll need to specify both flags to achieve same effect.un

**Comment by kevgs** [comment]
What plugin should do?

File: sql/sys_vars.cc (hunk 3 above)
Comment (midenok) [reply sql/sys_vars.cc:396]: You display internal mechansims to UI. Will **2** be ever used by user??

**Comment by midenok** [comment]
What it does now in `am/forced_vers`.

**Comment by midenok** [comment]
I'll finish it.

**Pull request closed without merging**